aiperf/common/config/user_config.py

```diff
@@ -293,3 +293,18 @@ def _get_artifact_stimulus(self) -> str:
     def timing_mode(self) -> TimingMode:
         """Get the timing mode based on the user config."""
         return self._timing_mode
+
+    @model_validator(mode="after")
+    def validate_multi_turn_options(self) -> Self:
+        """Validate multi-turn options."""
+        # Multi-turn validation: only one of request_count or num_sessions should be set
+        if (
+            "request_count" in self.loadgen.model_fields_set
+            and "num" in self.input.conversation.model_fields_set
+        ):
+            raise ValueError(
```

Comment:
wait, wont this break anybody using --conversation-num to mean --num-dataset-entries as well? I really think they need to be split, but people are already using it! grrrr